mava/wrappers/jaxmarl.py

```diff
@@ -167,35 +167,34 @@ def __init__(
         self,
         env: MultiAgentEnv,
         has_global_state: bool,
-        timelimit: int,
+        time_limit: int = -1,
     ) -> None:
         """Initialize the JaxMarlWrapper.
 
         Args:
         ----
         - env: The JaxMarl environment to wrap.
         - has_global_state: Whether the environment has global state.
-        - timelimit: The time limit for each episode.
-
+        - time_limit: The time limit for each episode.
         """
         # Check that all specs are the same as we only support homogeneous environments, for now ;)
         homogenous_error = (
             f"Mava only supports environments with homogeneous agents, "
             f"but you tried to use {env} which is not homogeneous."
         )
         assert is_homogenous(env), homogenous_error
+        assert time_limit > 0, f"Time limit must be greater than 0, got {time_limit}"
```

Comment:
why can't we just leave time_limit a mendatory attribute to fill and by that it will automatically raise an error without this assert?

Reply:
Good question. If you look in `make_env.py` when we create a jaxmarl env we don't pass the `time_limit` because each env has a set time limit - for mabrax its 1000 and so we set this in the mabrax constructor when we call `super` but we don't want it to be required to pass this in to the mabrax wrapper. So we need to keep all the JaxMarlWrappers the same that's why we have a default here (to make it optional) but we want all the children to set the value, that's why we have the assert because the default is -1, does that make sense?

Reply:
Yeah totally makes sense :raised_hands: can you just add a comment based on what you said now so it will not raise confusion